src/main/java/uk/gov/hmcts/reform/preapi/repositories/CaseRepository.java

```diff
@@ -34,4 +36,7 @@ Page<Case> searchCasesBy(
     Optional<Case> findByIdAndDeletedAtIsNull(UUID id);
 
     List<Case> findAllByReferenceAndCourt_Id(String caseReference, UUID courtId);
+
+    List<Case> findByStateAndClosedAtBefore(CaseState state, Timestamp date);
```

Comment:
Just so it matches the other method names- this is a magic method generated by JPA. The less they differ from each other the better (just if issues arise in the future). 
```suggestion
    List<Case> findAllByStateAndClosedAtBefore(CaseState state, Timestamp date);
```